docs/repositories-configure/code-patterns.md

```diff
@@ -64,7 +64,6 @@ Importing tool and pattern configurations from another repository can help you b
 !!! important
     Consider the following when importing tool and pattern configurations:
 
-    -   **Permissions:** You must have admin permissions on the source repository.
```

Comment:
@machadoit, please review the rationale for completely removing this line in this commit message: https://github.com/codacy/docs/pull/397/commits/857f9d780a455c9e6e7c139b94801c95e37b7319